JPEG_85_OPTIM_YCBCR;failed to seed tile z8,x120,y150: wms request for tileset pnoa_photo_JPEG_85_OPTIM_YCBCR returned an unsupported format: <?xml version='1,0' encoding="UTF-8" standalone="no" ?> <!DOCTYPE ServiceExceptionReport SYSTEM "http://schemas,opengis,net/wms/1,1,1/exception_1_1_1,dtd"> <ServiceExceptionReport version="1,1,1"> <ServiceException> msDrawMap(): WMS connection error, Failed to draw WMS layer named &#39;pnoa_fondo_peninsula&#39;, This most likely happened because the remote WMS server returned an invalid image, and XML exception or another unexpected result in response to the GetMap request, Also check and make sure that the layer&#39;s connection URL is valid, drawGDAL(): Unable to access file, GDALDatasetRasterIO() failed: Error while reading row 2463: libpng: Read Error </ServiceException> </ServiceExceptionReport> 36,6;129,2;129,2
JPEG_95_OPTIM_YCBCR;failed to seed tile z8,x120,y150: wms request for tileset pnoa_photo_JPEG_95_OPTIM_YCBCR returned an unsupported format: <?xml version='1,0' encoding="UTF-8" standalone="no" ?> <!DOCTYPE ServiceExceptionReport SYSTEM "http://schemas,opengis,net/wms/1,1,1/exception_1_1_1,dtd"> <ServiceExceptionReport version="1,1,1"> <ServiceException> msDrawMap(): WMS connection error, Failed to draw WMS layer named &#39;pnoa_fondo_peninsula&#39;, This most likely happened because the remote WMS server returned an invalid image, and XML exception or another unexpected result in response to the GetMap request, Also check and make sure that the layer&#39;s connection URL is valid, drawGDAL(): Unable to access file, GDALDatasetRasterIO() failed: Error while reading row 2463: libpng: Read Error </ServiceException> </ServiceExceptionReport> 34,6;136,6;136,6
PNGQ_BEST;failed to seed tile z8,x120,y150: wms request for tileset pnoa_photo_PNGQ_BEST returned an unsupported format: <?xml version='1,0' encoding="UTF-8" standalone="no" ?> <!DOCTYPE ServiceExceptionReport SYSTEM "http://schemas,opengis,net/wms/1,1,1/exception_1_1_1,dtd"> <ServiceExceptionReport version="1,1,1"> <ServiceException> msDrawMap(): WMS connection error, Failed to draw WMS layer named &#39;pnoa_fondo_peninsula&#39;, This most likely happened because the remote WMS server returned an invalid image, and XML exception or another unexpected result in response to the GetMap request, Also check and make sure that the layer&#39;s connection URL is valid, drawGDAL(): Unable to access file, GDALDatasetRasterIO() failed: Error while reading row 2463: libpng: Read Error </ServiceException> </ServiceExceptionReport> 38,4;123,0;123,0
PNGQ_FAST;failed to seed tile z8,x120,y150: wms request for tileset pnoa_photo_PNGQ_FAST returned an unsupported format: <?xml version='1,0' encoding="UTF-8" standalone="no" ?> <!DOCTYPE ServiceExceptionReport SYSTEM "http://schemas,opengis,net/wms/1,1,1/exception_1_1_1,dtd"> <ServiceExceptionReport version="1,1,1"> <ServiceException> msDrawMap(): WMS connection error, Failed to draw WMS layer named &#39;pnoa_fondo_peninsula&#39;, This most likely happened because the remote WMS server returned an invalid image, and XML exception or another unexpected result in response to the GetMap request, Also check and make sure that the layer&#39;s connection URL is valid, drawGDAL(): Unable to access file, GDALDatasetRasterIO() failed: Error while reading row 2463: libpng: Read Error </ServiceException> </ServiceExceptionReport> 37,3;126,6;126,6
JPEG_BEST_OPTIM_YCBCR;failed to seed tile z8,x120,y150: wms request for tileset pnoa_photo_JPEG_BEST_OPTIM_YCBCR returned an unsupported format: <?xml version='1,0' encoding="UTF-8" standalone="no" ?> <!DOCTYPE ServiceExceptionReport SYSTEM "http://schemas,opengis,net/wms/1,1,1/exception_1_1_1,dtd"> <ServiceExceptionReport version="1,1,1"> <ServiceException> msDrawMap(): WMS connection error, Failed to draw WMS layer named &#39;pnoa_fondo_peninsula&#39;, This most likely happened because the remote WMS server returned an invalid image, and XML exception or another unexpected result in response to the GetMap request, Also check and make sure that the layer&#39;s connection URL is valid, drawGDAL(): Unable to access file, GDALDatasetRasterIO() failed: Error while reading row 2463: libpng: Read Error </ServiceException> </ServiceExceptionReport> 33,7;140,4;140,4
JPEG_BEST_NOOPT_YCBCR;failed to seed tile z8,x120,y150: wms request for tileset pnoa_photo_JPEG_BEST_NOOPT_YCBCR returned an unsupported format: <?xml version='1,0' encoding="UTF-8" standalone="no" ?> <!DOCTYPE ServiceExceptionReport SYSTEM "http://schemas,opengis,net/wms/1,1,1/exception_1_1_1,dtd"> <ServiceExceptionReport version="1,1,1"> <ServiceException> msDrawMap(): WMS connection error, Failed to draw WMS layer named &#39;pnoa_fondo_peninsula&#39;, This most likely happened because the remote WMS server returned an invalid image, and XML exception or another unexpected result in response to the GetMap request, Also check and make sure that the layer&#39;s connection URL is valid, drawGDAL(): Unable to access file, GDALDatasetRasterIO() failed: Error while reading row 2463: libpng: Read Error </ServiceException> </ServiceExceptionReport> 34,5;136,9;136,9
PNG_BEST;failed to seed tile z8,x120,y150: wms request for tileset pnoa_photo_PNG_BEST returned an unsupported format: <?xml version='1,0' encoding="UTF-8" standalone="no" ?> <!DOCTYPE ServiceExceptionReport SYSTEM "http://schemas,opengis,net/wms/1,1,1/exception_1_1_1,dtd"> <ServiceExceptionReport version="1,1,1"> <ServiceException> msDrawMap(): WMS connection error, Failed to draw WMS layer named &#39;pnoa_fondo_peninsula&#39;, This most likely happened because the remote WMS server returned an invalid image, and XML exception or another unexpected result in response to the GetMap request, Also check and make sure that the layer&#39;s connection URL is valid, drawGDAL(): Unable to access file, GDALDatasetRasterIO() failed: Error while reading row 2463: libpng: Read Error </ServiceException> </ServiceExceptionReport> 35,5;133,1;133,1
PNG_FAST;failed to seed tile z8,x120,y150: wms request for tileset pnoa_photo_PNG_FAST returned an unsupported format: <?xml version='1,0' encoding="UTF-8" standalone="no" ?> <!DOCTYPE ServiceExceptionReport SYSTEM "http://schemas,opengis,net/wms/1,1,1/exception_1_1_1,dtd"> <ServiceExceptionReport version="1,1,1"> <ServiceException> msDrawMap(): WMS connection error, Failed to draw WMS layer named &#39;pnoa_fondo_peninsula&#39;, This most likely happened because the remote WMS server returned an invalid image, and XML exception or another unexpected result in response to the GetMap request, Also check and make sure that the layer&#39;s connection URL is valid, drawGDAL(): Unable to access file, GDALDatasetRasterIO() failed: Error while reading row 2463: libpng: Read Error </ServiceException> </ServiceExceptionReport> 16,3;289,7;289,7
